CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Reconcile redundant focus styles with U.S. Web Design System. ([#302](https://github.com/18F/identity-style-guide/pull/302))
 - Remove documentation site styles from design system artifact. ([#305](https://github.com/18F/identity-style-guide/pull/305))
 - Reduce size of SVG images. ([#307](https://github.com/18F/identity-style-guide/pull/307))
+- Incorporate mfa icon styles, which were previously added to the identity-idp. ([#309](https://github.com/18F/identity-style-guide/pull/309))
```

Comment:
This might fit better under an "Improvements" or "Bug Fixes" section, since it's not really a performance optimization. Could we describe what the change is. IIRC it's improving/increasing the spacing around the icon?